### ISSUE TITLE
Emoji Classification using OpenCV

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/ML-ProjectKart#49]
Hey LGM-SoC participants, good to see you.

This project will help in classify an image according to its visual content and then convert the label output to emoji output It classifies by collecting the user's photos.

Dataset : https://www.kaggle.com/msambare/fer2013

If you like to work on this issue, follow the given guidelines for code as well as for the README. You can try to compare it using different algorithms.

Enjoy your open source journey!😎

**Comment by noviicee** [comment]
Hey @prathimacode-hub 
Can I work on this under LGM-SOC'21 ?

**Comment by noviicee** [comment]
@prathimacode-hub 
Can I take this up as a part of lgm-soc'21?

**Comment by prathimacode-hub** [comment]
You can take up only 1 issue at a time. @noviicee

**Comment by noviicee** [comment]
Ok. You can assign me with this. Will wait for the other one then.

> You can take up only 1 issue at a time. @noviicee

**Comment by prathimacode-hub** [comment]
Ok, good decision. I had assigned this issue to you. @noviicee

**Comment by prathimacode-hub** [comment]
You will be working on model creation right? @noviicee

**Comment by prathimacode-hub** [comment]
There is no update from you since the issue is assigned to you. Any progress? @noviicee

**Comment by noviicee** [comment]
> There is no update from you since the issue is assigned to you. Any progress? @noviicee

Sorry for the delay..actually my exams are going on, that's why I am not active.
For the issue,  I was thinking of providing an image as input and it will be converted to corresponding emoji. Is there anything specific that needs to be done here..?

**Comment by prathimacode-hub** [comment]
No. it's the same. You will be working on Model Creation? When you take up a task, you should provide full details. And also exams are there, that's considered and it's important. So, it's better you give timely updates. @noviicee

**Comment by noviicee** [comment]
Yes, I understand. Will be updating soon :v:

**Comment by noviicee** [comment]
Hey @prathimacode-hub
I am facing health issues from the past few days. I guess I won't be able to make a PR soon. 
If anybody is interested to take up the issue, they make take it up!
#LGM-SoC participants

**Comment by prathimacode-hub** [comment]
Sure, thanks for letting me know. Health comes first and is of top most priority. I pray for your speedy recovery. Get well soon. @noviicee

**Comment by abhisheks008** [comment]
Hey @prathimacode-hub , I am up for this issue and I want to contribute in this.

Name of the issue : Emoji Classification using OpenCV
Issue #49

- [x] LGMSOC Participant

Approach : I will be solving this issue using the opencv library, apart from that I will be also using the libraries such as Tensorflow, Keras, seaborn and many more basic libraries. These are the essential components which I will be using to solve this complicated issue.

Description : The project will be classifying all the images given in the dataset, will be trained by the dataset and according to the training of the dataset it will provide the prediction of the emojis based on the dataset.

Task Taken : Developing the Classification and Prediction Model of this project.

Please assign me this issue.

**Comment by prathimacode-hub** [comment]
Sure, this issue is assigned to you. You can start working on it. @abhisheks008

**Comment by abhisheks008** [comment]
> Sure, this issue is assigned to you. You can start working on it. @abhisheks008

How much time will I get for this project to be done?
